arviz/tests/test_diagnostics.py

```diff
@@ -4,6 +4,7 @@
 import numpy as np
 from numpy.testing import assert_almost_equal, assert_array_almost_equal
 import pandas as pd
+import scipy.stats as st
```

Comment:
`from scipy.stats import circstd`? I may have missed some call to st, but it looks like only circstd is used